tests/validation_suite/work_order_tests/test_work_order_success.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import logging
+import json
+from automation_framework.utilities.process_api_request import \
+                                    process_api_request
+import automation_framework.work_order_submit.work_order_submit_utility as \
+                                                             wo_utility
+import automation_framework.work_order_get_result. \
+        work_order_get_result_utility as wo_get_result
+from automation_framework.utilities.request_args import TestStep
+
+logger = logging.getLogger(__name__)
+
+
+def test_work_order_success(setup_config):
+    """ Testing work order request with all valid parameter values. """
+
+    # retrieve values from conftest session fixture
+    worker_obj = setup_config[0]
+    uri_client = setup_config[1]
+    private_key = setup_config[2]
+    err_cd = setup_config[3]
+
+    # input file name
+    input_json_file = 'work_order_tests/input/work_order_success.json'
+    # input type - file, string or object
+    input_type = 'file'
+    # output filename
+    output_json_file_name = 'work_order_success'
+    # tamper parameters
+    tamper = {"params": {}}
+    # request method to be used when processing object input type
+    request_method = ""
+
+    # expected response
+    check_submit = {"error": {"code": 5}}
+    check_result = '''{"result": {"workOrderId": "", "workloadId": "",
+                       "workerId": "", "requesterId": "", "workerNonce": "",
+                       "workerSignature": "", "outData": ""}}'''
+    check_get_result = json.loads(check_result)
+
+    request_tup = (input_json_file, input_type, tamper, output_json_file_name,
+                   uri_client, request_method, worker_obj, private_key, err_cd,
+                   check_submit, check_get_result)
+    response_tup = process_api_request(request_tup)
+
```

Comment:
After submitting api request, do the validation.

Reply:
Done.